data-sets-tests/src/test/java/org/zowe/data/sets/tests/AbstractDataSetsIntegrationTest.java

```diff
@@ -102,4 +103,8 @@ static Response deleteDataSet(String dataSetName) {
     static String getDataSetMemberPath(String pds, String member) {
         return pds + "(" + member + ")";
     }
+    
+    static Response putDataSetRename(String oldDataSetName, DataSetRenameRequest body) {
+        return RestAssured.given().contentType("application/json").body(body).when().put(oldDataSetName + "/rename");
+    }
```

Comment:
This code doesn't seem to be shared, so should be pushed down into the sub class

Reply:
moved down as recommended